pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Describe your changes
+Please include a summary of the change and which issue is fixed.
+
+## Issue ticket number and link
+
+#### Checklist before requesting a review
+#### General
+- [ ] I have performed a self-review of my code.
+- [ ] Code changes are considered complete for the given story (i.e., the comments from code review are addressed)
+- [ ] The commit title follows conventional commit standards.
+- [ ] I have put the PR link into the planner.
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes.
+- [ ] Code builds cleanly without any errors or warnings
+ 
+#### Code Quality
+- [ ] I have run linting and there were no errors
+- [ ] The changes have been reviewed for accessibility compliance.
+ 
+#### Testing
+- [ ] New functionality is tested
+- [ ] All unit and integration tests passed
+- [ ] The changes have been reviewed for accessibility compliance.
+ 
+#### Deployment
+- [ ] Documentation has been updated where needed
+- [ ] Any required database migrations have been added and tested
```

Comment:
no db migrations in this repo

Reply:
oops missed this when trying to make it the same as the backend. I'll fix that